[Merged by Bors] - docs: updated authentication with LDAP section (#406)
stackabletech/nifi-operator: docs/modules/ROOT/pages/usage.adoc

The default setting is to only provision a single user with administrative privileges.
You need to specify the username and password of the user.
Additional users can not be added.

[#authentication-ldap]
=== LDAP

NiFi supports xref:home:concepts:authentication.adoc[authentication] of users against an LDAP server. This requires setting up an xref:home:concepts:authentication.adoc#authenticationclass[AuthenticationClass] for the LDAP server.

> The AuthenticationClass is then referenced in the NifiCluster resource like so:

Reviewer: ```suggestion
The AuthenticationClass is then referenced in the NifiCluster resource as follows:
```
